www-apps/radarr/files/radarr.init

```diff
@@ -0,0 +1,48 @@
+#!/sbin/openrc-run
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+depend() {
+        need net
+}
+
+run_dir="/var/run/radarr"
+pidfile="$run_dir/radarr.pid"
+etc_dir="/etc/radarr"
+var_dir="/var/lib/radarr"
+log_dir="/var/log/radarr"
+log_file="radarr.log"
+program="$APP_DIR/Radarr"
+
+check_config() {
+    
+    if [ ! -d "${run_dir}" ]; then
+            mkdir "${run_dir}"
+    fi
+
+    # Permission stuff. Should ensure that the daemon user always have write permissions.
+    # Quick and ugly but should get the job done.
+
+    chown -R ${USER}:${GROUP} "${run_dir}"
+        chown -R ${USER}.${GROUP} "${var_dir}"
+    chown -R ${USER}.${GROUP} "${etc_dir}"
+}
+
+start() {
+    check_config
+
+        ebegin "Starting Radarr"
```

Comment:
indentation looks wrong here and in a few places

Reply:
done, thanks for pointing that out!